tests/stress/cases/modules/loadbalancer/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "layer0_load_balancer" "base" {
```

Comment:
I think with some fancy interpolation, we can reduce everything into `cases/modules` into a single, flexible terraform configuration. 


A simple approach is to change the input variables for environment-specific entities from flat numbers to per-environment numbers:
```
variable "num_environments" {
    type = "int"
    default = 1
}

variable "services_per_environment" {
  type = "int"
  default = 1
}

resource "layer0_environment" "base" {
    ...
    count = "${var.num_environments}"
}

resource "layer0_service" "base" {
    ...
    environment_id = "${layer0_environment.base[count.index % var.num_environments].id}"
    count          = "${var.num_environments * var.services_per_environment}"
}
```

The name of the tests could potentially add some more detail:
```
func Benchmark_25ServicesAcross5Environments(b *testing.B) {
	benchmarkStress(b, 5, 5, 0, 0)
}
```

_edit - I think a struct could make this even more explicit_
```
func Benchmark_25ServicesAcross5Environments(b *testing.B) {
    benchmarkStress(b, BenchmarkTestCase{
        Environments:           5,
        ServicesPerEnvironment: 5,
    })
}
```